extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -406,6 +404,78 @@ angular.module('oppia').component('svgFilenameEditor', {
         };
       };
 
+      var loadGroupedObject = function(objId, obj, groupedObjects) {
+        // Checks the first five characters of the id.
+        if (objId.slice(0, 5) === 'group') {
```

Comment:
```suggestion
        if (objId.startsWith('group')) {
```
Would this work here? It looks cleaner.

Reply:
Done.